README.md

```diff
@@ -241,9 +241,19 @@ Pilot instance in the ingress namespace. This will be fixed in future releases.
 Note that running a dedicated Pilot for ingress/gateways is supported and recommended for very large sites, 
 but in the case of K8S ingress it is currently required.
 
+```bash
+    iop istio-ingress istio-ingress $IBASE/istio-installer/gateways/istio-ingress  --set global.istioNamespace=istio-master
```

Comment:
@costinm This is the case you described above, right? So it should not work? Actually, it didn't :( We would have to unset istioNamespace and install another pilot?

Reply:
Sorry :-), I can try it again. 

Try it first with istioNamespace=istio-control ( that's what I tested the most ). And logs/more info on what didn't work - usually it's some config that is not properly set.